components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -811,6 +811,12 @@ class CLPConfig(BaseModel):
 
     _os_release_file_path: pathlib.Path = PrivateAttr(default=OS_RELEASE_FILE_PATH)
 
+    @validator("aws_config_directory", pre=True)
+    def validate_aws_config_directory(cls, value):
```

Comment:
for consistency with other validators, i prefixed this with `validate` though it does not really validate the path (the actual valuation is done in `validate_aws_config_dir()`)

we may rename the method as `expand_profile_user_home` if the convention does not need to be followed

Reply:
sounds good. let me double check if the `pre=True` is still needed for good in that case